modules/services/agentless-scanning/organizational.tf

```diff
@@ -131,14 +135,18 @@ resource "aws_cloudformation_stack_set_instance" "scanning_role_stackset_instanc
 # stackset to deploy resources for agentless scanning in management account
 resource "aws_cloudformation_stack_set" "mgmt_acc_resources_stackset" {
   count      = var.is_organizational ? 1 : 0
-  depends_on = [aws_iam_role.agentless]
+  depends_on = [aws_iam_role.scanning]
 
   name                    = join("-", [var.name, "ScanningKmsMgmtAcc"])
   tags                    = var.tags
   permission_model        = "SELF_MANAGED"
   capabilities            = ["CAPABILITY_NAMED_IAM"]
   administration_role_arn = var.stackset_admin_role_arn
 
+  lifecycle {
+    ignore_changes = [administration_role_arn]
+  }
+
   template_body = <<TEMPLATE
 Resources:
   AgentlessScanningKmsPrimaryKey:
```

Comment:
nit: can we please update the `Description` for `AgentlessScanningKmsPrimaryKey` in this stackset as well?

Reply:
updated